java/jscip/Scip.java

```diff
@@ -464,4 +464,30 @@ public double getGap()
    {
       return SCIPJNI.SCIPgetGap(_scipptr);
    }
+
+   /** wraps SCIPsetStaticErrorPrintingMessagehdlr() */
+   public static void setStaticErrorPrintingMessagehdlr(MessageHandler messagehdlr) {
+      SCIPJNI.SCIPsetStaticErrorPrintingMessagehdlr(MessageHandler.getPtr(messagehdlr));
+   }
+
+   /** wraps SCIPsetMessagehdlr() */
+   public void setMessagehdlr(MessageHandler messagehdlr) {
+      CHECK_RETCODE(SCIPJNI.SCIPsetMessagehdlr(_scipptr, MessageHandler.getPtr(messagehdlr)));
```

Comment:
Minor whitespace change: `CHECK_RETCODE( SCIPJNI...`

Reply:
OK, I will change that, for consistency with the existing code.

Reply:
Done in commit a1aee50f31613852e804709b027599d405567fe2.